noxfile.py

```diff
@@ -110,10 +110,8 @@ def system(session):
 
     # Install all test dependencies, then install this package into the
     # virtualenv's dist-packages.
-    session.install("mock", "pytest")
-
-    session.install("-e", "test_utils")
-    session.install("-e", "psutil")
+    session.install("mock", "pytest", "psutil")
+    session.install("git+https://github.com/googleapis/python-test-utils")
```

Comment:
Nice, thanks! :heart: 

BTW, is now time to remove the local `test_utils` directory? (I can do it, in a separate PR)